src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/Unsafe.cs

```diff
@@ -238,6 +238,24 @@ public static bool AreSame<T>([AllowNull] ref T left, [AllowNull] ref T right)
             // ret
         }
 
+        /// <summary>
+        /// Reinterprets the given reference as a reference to a value of type <typeparamref name="TTo"/>.
```

Comment:
nit: the method doesn't take or return a reference but the doc comment implies it does

Reply:
Fixed.